playground/mocks/config/responseConfig.js

```diff
@@ -15,7 +15,8 @@ const idx = {
 
   '/idp/idx/introspect': [
     'identify',
-    // 'error-401-invalid-otp-passcode',
+    // 'error-401-challenge-invalid-otp-passcode',
+    // 'error-401-enroll-invalid-otp-passcode',
```

Comment:
Only added `challenge` vs `enroll`
@pradeepdewda-okta I don't think we required sample for each authenticators but if so let me know. I can add separate one for Email/Phone...Not sure about mobile otp.